core/dbt/contracts/project.py

```diff
@@ -263,7 +263,7 @@ class ProfileConfig(HyphenatedDbtClassMixin, Replaceable):
     threads: int
     # TODO: make this a dynamic union of some kind?
     credentials: Optional[Dict[str, Any]]
-
+    manage_schema: Optional[bool]
```

Comment:
This merges the manage_schema configuration with the profile, but I think it might be nicer to move this configuration to a top-level `manage_targets` configuration (list of targets to enable schema management for).
The difference is having
```
config:
  manage_targets: [test, prod]
```
vs
```
snowflake:
  outputs:
    dev:
      manage_schema: true
```

Not sure about this yet, I also don't know if target names are unique per driver or per profile config file...

Reply:
If I understand this correct, your code change is done in project, allowing the user to specify this in `dbt_project.yml` eventually, while your comment talks about the profile. 
- Do I understand correctly?
- I think doing this on a project level makes it no longer possible to have different configs for dev and pro, for example.

Reply:
The second option you mention in your comment is clearer to me. When looking at dev, you know its config right away, without moving to another place in the yaml to check the manage_targets variable.

What pros do you see with your first option?